projects/nginx/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eu
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+./auto/configure --with-cc-opt="-DNGX_DEBUG_MALLOC -fsanitize=address"        \
+--with-ld-opt="-fsanitize=address" --add-module=$SRC/conf-module \
+--with-debug --prefix=$OUT
+
+make -j$(nproc)
+make install
+
+$CXX $CXXFLAGS -std=c++11 -Isrc/ \
+     $SRC/conf_fuzzer.cc -o $OUT/conf_fuzzer \
+     -lFuzzingEngine
+
+cp $SRC/*.dict $SRC/*.options $OUT/
```

Comment:
Please also consider adding a corpus. 
https://github.com/google/oss-fuzz/blob/master/docs/new_project_guide.md#seed-corpus
It does not have to be there, but it may in some cases make a huge difference in results.